state/cleanup.go

```diff
@@ -237,6 +240,66 @@ func (st *State) cleanupRelationSettings(prefix string) error {
 	return nil
 }
 
+func (st *State) cleanupForceDestroyedRelation(prefix string) (err error) {
+	relation, err := st.KeyRelation(prefix)
+	if err != nil {
+		return errors.Annotatef(err, "getting relation %q", prefix)
+	}
+	scopes, closer := st.db().GetCollection(relationScopesC)
+	defer closer()
+
+	sel := bson.D{{"_id", bson.D{{"$regex", "^" + st.docID(relation.globalScope())}}}}
```

Comment:
To be safe should we add a "#" (I think that's the separator) to the end of the regexp expression?

Reply:
Yeah, good call.